packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -0,0 +1,415 @@
+                setStatus({ msg: data.error.message });
+                setFormState({ ...form_state, ...{ is_btn_loading: false } });
+                setSubmitting(false);
+                return;
+            }
+
+            const get_settings = WS.authorized.storage.getSettings();
+
+            if (get_settings.error) {
+                setAPIInitialLoadError(get_settings.error.message);
+                setSubmitting(false);
+                return;
+            }
+            const { address_line_1, address_line_2, address_city, address_state, address_postcode } = get_settings;
+
+            setFormValues({
+                address_line_1,
+                address_line_2,
+                address_city,
+                address_state,
+                address_postcode,
+            });
+            setIsLoading(false);
+
+            // upload files
+            try {
+                const api_response = await file_uploader_ref?.current?.upload();
+                if (api_response?.warning) {
+                    setStatus({ msg: api_response?.message });
+                    setFormState({ ...form_state, ...{ is_btn_loading: false } });
+                    return;
+                }
+
+                const get_account_status = WS.authorized.storage.getAccountStatus();
+
+                if (get_account_status.error) {
+                    setAPIInitialLoadError(get_account_status.error.message);
+                    setSubmitting(false);
+                    return;
+                }
+
+                setFormState({
+                    ...form_state,
+                    ...{ is_submit_success: true, is_btn_loading: false },
+                });
+
+                const { identity, needs_verification } = get_account_status.authentication;
+                const has_poi = !(identity && identity.status === 'none');
+                const needs_poi = needs_verification.length && needs_verification.includes('identity');
+                onSubmit?.(has_poi);
+                removeNotificationMessage({ key: 'authenticate' });
+                removeNotificationByKey({ key: 'authenticate' });
+                removeNotificationMessage({ key: 'needs_poa' });
+                removeNotificationByKey({ key: 'needs_poa' });
+                removeNotificationMessage({ key: 'poa_expired' });
+                removeNotificationByKey({ key: 'poa_expired' });
+                if (needs_poi) {
+                    addNotificationByKey('needs_poi');
+                }
+            } catch (error) {
+                if (isServerError(error)) {
+                    setStatus({ msg: error.message });
+                    setFormState({ ...form_state, ...{ is_btn_loading: false } });
+                }
+            } finally {
+                setSubmitting(false);
+                setFormState({ ...form_state, ...{ is_btn_loading: false } });
+            }
+            if (is_qualified_for_cfd_modal && typeof step_index !== 'undefined') {
```

Comment:
what is actullay  is_qualified_for_cfd_modal mean? 
Does it means it is from cfd modal?  if yes, can we rewrite it to something like is_from_mt5_modal?

Reply:
yes, it is for cfd modal. 
I apply here the same naming as we did for rendering for example PersonalDetails,
we use there 'is_qualified_for_idv', 'is_qualified_for_onfido' (now  it is 'is_rendered_for_onfido', refactoring it in this PR https://github.com/binary-com/deriv-app/pull/8884)

Reply:
so basically it means, that component is rendered for some feature, but not comes from some other feature

Reply:
"Is_qualified_for_cfd_modal" implies it is based on a  particular status,  could we consider a better name?

Reply:
Alright, @amina-deriv, will change the name